lib/wallet-scoring.js

```diff
@@ -3,25 +3,32 @@ const _ = require('lodash/fp')
 const argv = require('minimist')(process.argv.slice(2))
 
 function loadWalletScoring (settings) {
-  if (_.isNil(argv.mockScoring)) {
-    throw new Error('No wallet scoring API set!')
-  }
-  const pluginCode = argv.mockScoring ? 'mock-scoring' : ''
+  const pluginCode = argv.mockScoring ? 'mock-scoring' : 'ciphertrace'
```

Comment:
We should use the same plugin interface as the other third parties. "None" is a possibility here as not all operators will configure this.